### PR TITLE
Update example and typos

### PR DESCRIPTION
**Pull request by ramirogm**
- `metadata.event.type` and `metadata.event.action` had invalid values according to the enum definition
- `code` is a `number` not `string`
- `spanId` should be `parentSpanId`

### CODE REVIEW

File: mojaloop-technical-overview/event-framework/README.md [hunk 1]
```diff
@@ -47,8 +47,8 @@ The purpose of the Event Framework is to provide a standard unified architecture
     "metadata": {
         "event": {
             "id": "3920382d-f78c-4023-adf9-0d7a4a2a3a2f",
-            "type": "position",
-            "action": "commit",
+            "type": "log",
+            "action": "start",
```
Comment (mdebarros) [inline mojaloop-technical-overview/event-framework/README.md:51]: Note that this is not a valid action for an event type of `log`. See the data model specification below. 

Refer to https://github.com/mdebarros/documentation/tree/feature/%23767DesignEvent%2CAuditingandErrorFrameworkProposal/mojaloop-technical-overview/event-framework#3215-enum-eneventtype for details

File: mojaloop-technical-overview/event-framework/README.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ The purpose of the Event Framework is to provide a standard unified architecture
         "trace": {
             "service": "central-ledger-prepare-handler",
             "traceId": "bbd7b2c7-3978-408e-ae2e-a13012c47739",
-            "parentId": "4e3ce424-d611-417b-a7b3-44ba9bbc5840",
+            "parentSpanId": "4e3ce424-d611-417b-a7b3-44ba9bbc5840",
```
Comment (mdebarros) [inline mojaloop-technical-overview/event-framework/README.md:63]: Thanks for typo fix

File: mojaloop-technical-overview/event-framework/README.md [hunk 3]
```diff
@@ -80,7 +80,7 @@ TBD
 | from | string | Optional. If the value is not present in the destination, it means that the notification was generated by the connected node (server). |  |
 | to | string | Mandatory for the sender and optional in the destination. The sender can ommit the value of the domain. | |
 | pp | string | Optional for the sender, when is considered the identity of the session. Is mandatory in the destination if the identity of the originator is different of the identity of the from property. | |
-| metadata | object \<metadataType\> | Optional. The sender should avoid to use this property to transport any kind of content-related information, but merely data relevant to the context of the communication. Consider to define a new content type if there's a need to include more content information into the message. | |
+| metadata | object \<ObMetadataType\> | Optional. The sender should avoid to use this property to transport any kind of content-related information, but merely data relevant to the context of the communication. Consider to define a new content type if there's a need to include more content information into the message. | |
```
Comment (mdebarros) [inline mojaloop-technical-overview/event-framework/README.md:83]: Thanks for typo fix

File: mojaloop-technical-overview/event-framework/README.md [hunk 4]
```diff
@@ -108,7 +108,7 @@ TBD
 | Name | Type | Description | Example |
 | --- | --- | --- | --- |
 | status | enum \<EneventStatus\> | Mandatory. The id references the related message. | success |
-| code | string | Optional. The error code as per Mojaloop specification. | 2000 |
+| code | number | Optional. The error code as per Mojaloop specification. | 2000 |
```
Comment (mdebarros) [inline mojaloop-technical-overview/event-framework/README.md:111]: cool.

**Review by mdebarros — CHANGES_REQUESTED** [review]
See comments below.

File: mojaloop-technical-overview/event-framework/README.md (hunk 1 above)
Comment (ramirogm) [reply mojaloop-technical-overview/event-framework/README.md:51]: Ok, got it.

Comment (ramirogm) [reply mojaloop-technical-overview/event-framework/README.md:51]: Changed to trace/start

**Comment by ramirogm** [comment]
@mdebarros Fixed

**Pull request merged**